lib/lint.ml

```diff
@@ -118,6 +118,9 @@ module Check = struct
         | exn -> Lwt.fail exn
         end
 
+  let is_dune name =
+    OpamPackage.Name.equal name (OpamPackage.Name.of_string "dune")
```

Comment:
The compiler packages also trigger this one - cf. https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/6bac8b5b3330550680e20409e483074590e54b95/variant/(lint)

Reply:
Since dune is an optional tool for the compiler, something like `depopts: ["dune" { with-tools }]` might make sense for it, but it's a can of worms.